### ISSUE TITLE
Issue that does not publish sample buffers when used only audio

### ISSUE THREAD

**Issue by Jellabean07** [shogo4405/HaishinKit.swift#1027]
### Describe the bug

- If audio is used only, captureOutput cannot send a sample buffer because AVMixer's useSampleBuffer function always returns false.  
- The reason is that MediaSync is always fixed with video when only attachAudio is called
- Therefore, exception handling is required for audio only

```
func captureOutput(_ output: AVCaptureOutput, didOutput sampleBuffer: CMSampleBuffer, from connection: AVCaptureConnection) {
        
        guard mixer?.useSampleBuffer(sampleBuffer: sampleBuffer, mediaType: AVMediaType.audio) == true else {
            return
        }
        appendSampleBuffer(sampleBuffer)
    }
```

```
func useSampleBuffer(sampleBuffer: CMSampleBuffer, mediaType: AVMediaType) -> Bool {
        switch mediaSync {
        case .video:
            if mediaType == .audio {
                return !videoTimeStamp.seconds.isZero && videoTimeStamp.seconds <= sampleBuffer.presentationTimeStamp.seconds
            }
            if videoTimeStamp == CMTime.zero {
                videoTimeStamp = sampleBuffer.presentationTimeStamp
            }
            return true
        case .audio:
            if mediaType == .video {
                return !audioTimeStamp.seconds.isZero && audioTimeStamp.seconds <= sampleBuffer.presentationTimeStamp.seconds
            }
            if audioTimeStamp == CMTime.zero {
                audioTimeStamp = sampleBuffer.presentationTimeStamp
            }
            return true
        default:
            return true
        }
    }
```

### To Reproduce

Use only the attachAudio function and do not use attachVideo.

### Expected behavior

The sample buffer should be published when using audio only

### Version

1.2.6

**Comment by shogo4405** [comment]
Thank you for feedback. I fixed it [cd57a82](https://github.com/shogo4405/HaishinKit.swift/commit/cd57a82d55121ace0a91b06e2ba8ce03a7fabc0c).

**Comment by Jellabean07** [comment]
@shogo4405 Thank you for fixed it 
I'll look forward to the version being updated
